### DISCUSSION TITLE
Converting Old Text File to New Database Format?

### DISCUSSION THREAD

**Discussion by SarahSzabo** [chylex/Discord-History-Tracker#175] [General]
Hello, I have an old database viewer file which is an archive of old Discord servers that are no longer in existance.

Is there any way to convert these to the new database format?
I tried looking for settings for this in the Linux app, but no luck.

**Comment by chylex** [comment]
Not at the moment. A tool to do that could exist with some limitations, but I don't have time atm. https://github.com/chylex/Discord-History-Tracker/discussions/145#discussioncomment-1015290

**Reply by SarahSzabo** [reply]
I have programming skills, but I don't have the time either honestly, and I'm not familiar with the SQL notation, or database builds.
